clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/securitygroup/SecurityGroupLookupFactory.groovy

```diff
@@ -294,9 +295,10 @@ class SecurityGroupLookupFactory {
         List<TagDescription> currentTags = tagsResult.getTags()
         Collection<Tag> oldTags = new HashSet()
         // Filter Spinnaker specific tags, update to other tags might result in permission errors
+        def additionalTags = dynamicConfigService.getConfig(String.class, "aws.features.security-group.additional-tags","")
         currentTags.each {
           it ->
-            if (it.key.equals("Name") || description.tags.keySet().contains(it.key)) {
+            if (it.key.equals("Name") || description.tags?.keySet()?.contains(it.key) || additionalTags.contains(it.key)) {
```

Comment:
Can we get a real-world example of what would be in `additionalTags`? Is it really something that should be backed by the `DynamicConfigService` or should it be some config object that wraps `List<String>` (rather than just a plain `String`?

Just so I'm understanding this and the previous PR, the goal here is to only remove a subset of all potential tags on the security group. 

Effectively "Name" and any others that have been supplied as part of the description?

I assume that if there are policies governing special tags that would prevent removal, that they might also prevent any change?